cpu/Makefile.include.cortexm_common

```diff
@@ -15,6 +15,24 @@ export CFLAGS_OPT  ?= -Os
 export CFLAGS += $(CFLAGS_CPU) $(CFLAGS_LINK) $(CFLAGS_DBG) $(CFLAGS_OPT)
 
 export ASFLAGS += $(CFLAGS_CPU) $(CFLAGS_DBG)
+
+# If the FW_SLOTS flag is set, we will generate a linker script at compile-time
+# reflecting the FW_IMAGE_OFFSET and FW_IMAGE_LENGTH values defined in the
+# target project's Makefile.
+ifeq ($(FW_SLOTS),1)
+ifeq ($(FW_SLOT),1)
```

Comment:
What about `LINKER_SCRIPT = $(CPU_MODEL)_slot$(FW_SLOT).ld`. Then you don't limit it to 2 (it might be a future use case).